Rooms/masterBedroom.json

```diff
@@ -6,7 +6,7 @@
     "conditionals": {
         "boxesRoom": {
             "triggerSequence": ["boxes", "battery"],
-            "triggerVerbs": [["Examine", "Open"], ["Take"]],
+            "triggerVerbs": [["Examine", "Open"], ["Take", "Use"]],
```

Comment:
What happens if the player types "Use battery"? I think it will trigger this condition even though they don't actually use the battery.

use("battery") -> Triggers "except IndexError", sets item2 = None and numItems = 1
 if numItems = 1: 
---   if verbHelper("battery") -> prints("you have to pick it up") and returns True (which will step inside this if statement)
------      triggerConditionRoom("battery", "Use") -> Which will trigger this condition

Reply:
You are correct, forgot to go back and change that part. Added the fix.